raiden/api/rest.py

```diff
@@ -138,6 +144,7 @@
     ),
     ("/connections/<hexaddress:token_address>", ConnectionsResource),
     ("/connections", ConnectionsInfoResource),
+    ("/paymentsLumino/invoice", PaymentInvoiceResourceLumino),
```

Comment:
change the endpoint name, we should use /payments/invoice and not /paymentsLumino/invoice

Reply:
ok ready